x-pack/test/cloud_security_posture_functional/page_objects/findings_page.ts

```diff
@@ -113,11 +113,6 @@ export function FindingsPageProvider({ getService, getPageObjects }: FtrProvider
       return await retry.try(async () => {
         await testSubjects.click(actionTestSubject);
         await PageObjects.header.waitUntilLoadingHasFinished();
-        const result = await testSubjects.exists('createPackagePolicy_pageTitle');
-
-        if (!result) {
-          throw new Error('Integration installation page not found');
-        }
```

Comment:
Because the new button do not navigate to the create package policy page and in order to keep using the same framework for the new 3P prompt I had to either move this piece of logic outside, or pass "waitUntilElementIdExist" into the function. I ended up with moving this logic outside, not 100% sure that's the right decision here, it just felt a bit more straight forward, but its also less strict. let me know if you think the other way is better.